crux-bench/src/crux/bench.clj

```diff
@@ -117,8 +139,8 @@
       :crux.standalone/event-log-dir (str (io/file data-dir "eventlog/rocksdb"))
       :crux.standalone/event-log-kv-store 'crux.kv.rocksdb/kv})
 
-   "kafka-rocksdb"
-   (fn [data-dir]
+   #_"kafka-rocksdb"
```

Comment:
why commented out?

Reply:
Whoops - was running it locally and forgot to uncomment, thanks jon.